test-runner-frontend/src/Views/Dashboard.test.js

```diff
@@ -1,23 +1,49 @@
 import Dashboard from './Dashboard';
 import { render, fireEvent, waitFor, screen, getByRole, queryHelpers, getByTestId } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
+import fetchMock from 'jest-fetch-mock';
```

Comment:
do we need to enable this or does it work straight away?

Reply:
I found out I could make the tests using useContext() instead so we no longer use it, I already removed it.
But, I think we need to enable it